automatic/7zip.install/tools/chocolateyUninstall.ps1

```diff
@@ -4,6 +4,7 @@ $packageName = '7zip.install'
 
 $uninstalled = $false
 [array]$key = Get-UninstallRegistryKey -SoftwareName '7-zip*'
+$key = $key | Where-Object{ ! $_.DisplayName -Like '7-zip zs*' }
```

Comment:
We use lowercase for operators, also there is the operator `-notlike` that can also be used, and last the length of the line is not that much and you cane use piping directly here

```suggestion
[array]$key = Get-UninstallRegistryKey -SoftwareName '7-zip*' | Where-Object { $_.DisplayName -notlike '7-zip zs*' }
```

Reply:
Fixed...